### PR TITLE
Hungarian notation changed for member vars to pure camelcase

### PR DESCRIPTION
**Pull request by dasaldino1**

### CODE REVIEW

**Comment by lance** [comment]
@dasaldino1 I think it might be best to land https://github.com/Team8727/2023_Robot_Code/pull/19 first, then rebase your PR, since I'm not really sure how well Code Spaces will handle merge conflicts. (It may handle it just fine but 🤷). Once @WillHamel8727's PR is merged, I can help you with rebasing.

File: src/main/java/frc/robot/subsystems/Indications.java [hunk 1]
```diff
@@ -34,15 +33,15 @@ public RGB(int red, int green, int blue){
...
 
 
  public Indications() {
-   m_LedStrip = new AddressableLED(kSensors.ledPort);
-   m_LedData = new AddressableLEDBuffer(kSensors.ledLength);
-   m_LedStrip.setData(m_LedData);
-   m_LedStrip.start();
+   ledStrip = new AddressableLED(kSensors.ledPort);
+   ledData = new AddressableLEDBuffer(kSensors.ledLength);
+   ledStrip.setData(m_LedData);
```
Comment (bznerd) [inline src/main/java/frc/robot/subsystems/Indications.java:43]:
```suggestion
   ledStrip.setData(ledData);

```

**Comment by bznerd** [comment]
Consistency :partying_face:

**Review by bznerd — APPROVED** [review]
Big fan of this. We do need to revisit function and variable names for clarity, but that can happen when we do a documentation pass.

**Pull request merged**